### PR TITLE
add BERLIN asset

### PR DESCRIPTION
**Pull request by sirnicolaz**
## Description

Adding BERLIN token, and ERC20 IBC coin from chain EVMOS.

BERLIN is the token of Teledisko DAO, legally recognized as Teledisko DAO OÜ. Some information:

[Estonian Business Registry](https://ariregister.rik.ee/eng/company/16374990/teledisko-DAO-O%C3%9C?search_id=66cc896&pos=1)
[GitHub](https://github.com/TelediskoDAO/contracts)
[Discord](https://discord.gg/gVZsABrr)
[Website](https://www.teledisko.com)
Further information can be found in the [EVMOS governance proposal](https://wallet.keplr.app/chains/evmos/proposals/268) that allowed our token to become an IBC coin.

Adding pairs:

- BERLIN/OSMO
- BERLIN/USDC

### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is being registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry). Waiting for PR approval https://github.com/cosmos/chain-registry/pull/3890
- [x] Add asset to bottom of `zone_assets.json`.
- [x] The addition of the asset to the EVMOS dashboard has also been requested https://github.com/evmos/chain-token-registry/pull/88

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: 

BERLIN/OSMO
- [x] Ready -- A liquidity pool has been created. The pool ID is: 1487
Further liquidity (the equivalent around 100k USDC) will be added once we have confirmation that the pool is fine

BERLIN/USDC
- [x] Ready -- A liquidity pool has been created. The pool ID is: 1488
Further liquidity (around 100k USDC) will be added once we have confirmation that the pool is fine

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -2682,6 +2682,19 @@
         }
       ],
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "evmos",
+      "path": "transfer/channel-204/erc20/0x1cFc8f1FE8D5668BAFF2724547EcDbd6f013a280",
+      "base_denom": "erc20/0x1cFc8f1FE8D5668BAFF2724547EcDbd6f013a280",
+      "osmosis_verified": false,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2690]: please add "osmosis_unlisted": true

**Comment by JeremyParish69** [comment]
Did you mean to apply a swap fee of 2%?

**Comment by sirnicolaz** [comment]
> Did you mean to apply a swap fee of 2%?

I have applied the same that we applied for NEOK/OSMO back then. If you recommend to keep it lower, we can create new pools.

**Comment by JeremyParish69** [comment]
> I have applied the same that we applied for NEOK/OSMO back then. If you recommend to keep it lower, we can create new pools.

Ok, no, this is fine. Just wanted to make sure, It's just a common mistake to set the fee to 2% instead of 0.2%.

**Comment by JeremyParish69** [comment]
I'm having trouble validating the Deposit process. It seems the Evmos Portfolio site isn't loading. I'll try again later, but if it doesn't work then we'll need to find an alternative transfer interface.

**Comment by JeremyParish69** [comment]
Thanks. Listed now.

**Pull request merged**